src/css/index.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+.right{
+    overflow: hidden;
+    white-space: nowrap;
+  float:right;
+  display:inline-block;
+    border:1px solid red;
+}
+
+.left{
+    overflow: hidden;
+    white-space: nowrap;
+    display:inline-block;
+}
+
+.ul-inline{
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header{
+ display: inline-block;
+}
+
+.circle{
+  border-radius:50%;
+}
+
+.text-center{
+  margin:auto;
+  text-align: center;
+}
+
+.oval {
+  height: 269px;	width: 269px;	background-color: #FFFFFF;	box-shadow: 0 2px 12px 3px rgba(0,0,0,0.14);
+}
+
+.rectangle {
+  height: 45px;	width: 45px;	border: 3px solid #004853;
+}
+
+.pad-top-20px{
+  margin-bottom:40px;
+}
+
+.addie-hogan-addiehog {
```

Comment:
what is this style for?

Reply:
It is from the addie's css ,,,, just read through it to see what it does